python/llm/test/benchmark/check_results.py

```diff
@@ -20,26 +20,51 @@
 import sys
 import argparse
 import pandas as pd
+from omegaconf import OmegaConf
 
 def main():
     parser = argparse.ArgumentParser(description="check if the number of lines in html meets expectation")
+    parser.add_argument("-c", "--csv_file", type=str, dest="csv_name",
+                        help="name of csv", default="")
+    parser.add_argument("-y", "--yaml_file", type=str, dest="yaml_name",
```

Comment:
Can remove the default value and use `if args.yaml_file` to check?

Reply:
sure